packages/perseus/src/widgets/passage/__tests__/passage-markdown_test.jsx

```diff
@@ -352,14 +351,6 @@ describe.skip("passage markdown", () => {
         });
     });
 
-    describe("orders", () => {
```

Comment:
I know this is going to be controversial and I'm willing to change it back, but here's my thinking:

- Flow gets mad when you try to export things that aren't perfectly typed
- `rules` is only exported for tests and is a huge, mysterious object
- This was working because of a `$FlowFixMe` that, when removed, gave us better coverage and exposed some potential issues.
- This isn't _really_ a unit test, it's more a check you'd want in the logic. By that I mean SimpleMarkdown or PassageMarkdown should be checking if we're using the API correctly and throwing if not - checking in the development cycle, not the testing cycle.

Reply:
Here's the [commit](https://github.com/Khan/webapp/commit/93589dda5b25fc132c7a5cba3e87fd38604fb417) that introduced this test.  

These `rules` are basically a set of parsing functions and output functions (you can find the defaults [here](https://github.com/Khan/webapp/commit/93589dda5b25fc132c7a5cba3e87fd38604fb417)).

I think it'd be good to prioritize getting better typing around the `rules` in `SimpleMarkdown` and then in `passage-markdown.jsx` to avoid the risk of us re-introducing a bug like the original commit fixed, by accident.